### ISSUE TITLE
default port conflicts with jupyter notebook

### ISSUE THREAD

**Issue by greaber** [torrvision/crayon#31]
Crayon should use a more random default port number than 8888 since that conflicts with the default for jupyter notebook.

**Comment by bosr** [comment]
Yes, I agree, I made #38 for that purpose. We'll see what happens.
